scripts/check-external-layers-providers.js

```diff
@@ -54,9 +54,18 @@ function setupAxiosRetry() {
 }
 
 function compareResultByProvider(a, b) {
-    if (a['provider'] > b['provider']) {
+    if (a['provider'].toLowerCase() > b['provider'].toLowerCase()) {
         return 1
-    } else if (a['provider'] < b['provider']) {
+    } else if (a['provider'].toLowerCase() < b['provider'].toLowerCase()) {
+        return -1
+    }
+    return 0
```

Comment:
we could `compareCaseInsensitive(a['provider'], b['provider'])` here, so that we re-use the case insensitive test instead of writting it twice 😉